sargo/config.json

```diff
@@ -527,6 +527,79 @@
       "BoardConfigVendor": [],
       "device-vendor": [],
       "vendor-skip-files": []
+    },
+    "full": {
+      "overlays-dir": "",
+      "system-bytecode": [
+        "system/app/atfwd/atfwd.apk",
+        "system/app/ConferenceDialer/ConferenceDialer.apk",
+        "system/priv-app/GoogleDocumentsUIPrebuilt/GoogleDocumentsUIPrebuilt.apk",
```

Comment:
This and a bunch of apps below are Google Apps - which is what I meant when I said we aren't allowed to ship Google Apps anyway.

Reply:
Yes, I did check again and that seem to be apps that I added to the config.json after received compilation (CHAOSP + opengapps/aosp_build) errors about them. Obviously GoogleDocumentsUIPrebuilt.apk shouldn't be included, you're right.
I can't determine what ConferenceDialer.apk is and it doesn't seem to be AOSP-based so yes, this one may also be discarded.
But the atfwd.apk comes from Qualcomm and is a blob binary needed for WiFi Display/Miracast to work so I think it belongs there.

Anyway, this project is not shipping anything, it just keeps a list of which binary blobs to extract from publicly available zip archive files, to a local AOSP build tree. It's nowhere near the legal problem some custom ROMs had in the past by directly including the Google Apps binaries in the already built custom ROM.